frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -38,164 +38,127 @@ const Banner = ({ children }: TProps) => (
   <div className={styles.banner}>{children}</div>
 );
 
-type State = {
-  accountCode: accountAPI.AccountCode;
-  aopp?: aoppAPI.Aopp;
-}
-
-type Props = TranslateProps;
+const setAccountCodeDefault = (aopp: aoppAPI.Aopp | undefined, setAccountCode: React.Dispatch<React.SetStateAction<string>>) => {
```

Comment:
use newlines for arguments whenever there is something more complicated like this.